app/views/HealthIndicatorsLeaderBoard.scala.html

```diff
@@ -93,13 +93,33 @@ <h1>Health Indicator Leaderboard</h1>
         </table>
     </div>
 
+@*List.js configuration below >>>>> *@
     <script>
-        var options = {valueNames: ['repoName']};
+        var options = {valueNames: ['overallScore']};
         var indicatorsList = new List('leaderboard-display', options);
 
         var searchBox = document.getElementById("repo-name-search");
         indicatorsList.search(searchBox.value);
 
+        //Add a custom sorting function, as the default List.js sorting algorithm cannot handle negative numbers.
+        //Note: A Listener is required, as the List.js custom 'sortFunction' only auto-fires on page load, not on button click.
+        document.getElementById("overall-score-button").addEventListener("click", function (e) {
+            indicatorsList.sort('overallScore', {
+                order: e.target.classList.contains("desc") ? "desc" : "asc",
+                sortFunction: function(a, b) {
+                    var a = parseInt(a._values.overallScore)
+                    var b = parseInt(b._values.overallScore)
+                    if (a > b) {
```

Comment:
does it have to be exactly in the range of -1 to 1? if not, i guess you could just return a - b for more or less the same effect?